libc-top-half/musl/src/thread/pthread_condattr_setclock.c

```diff
@@ -1,9 +1,11 @@
 #include "pthread_impl.h"
 
+#include <common/clock.h>
+
 int pthread_condattr_setclock(pthread_condattr_t *a, clockid_t clk)
 {
-	if (clk < 0 || clk-2U < 2) return EINVAL;
+	if (clk->id < 0 || clk->id-2U < 2) return EINVAL;
 	a->__attr &= 0x80000000;
-	a->__attr |= clk;
+	a->__attr |= clk->id;
 	return 0;
```

Comment:
For changes to musl sources like this we wrap them in `__wasilibc_unmodified_upstream`

Reply:
updated